board/drivers/interrupts.h

```diff
@@ -53,9 +52,8 @@ void init_interrupts(bool check_rate_limit){
   }
 
   // Init timer 10 for a 1s interval
-  register_set_bits(&(RCC->APB1ENR), RCC_APB1ENR_TIM6EN);  // Enable interrupt timer peripheral
```

Comment:
Where/why did this go?

Reply:
It is a duplicate. This is already enabled in peripherals_init. Main peripheral clocks should be enabled only in one place imho. (If they are not tied to specific driver that is not used in every panda generation)

Reply:
I see, sounds good then